package.json

```diff
@@ -12,7 +12,8 @@
     "ip": "2.0.1",
     "node-polyfill-webpack-plugin": "^3.0.0",
     "vue-draggable-next": "^2.2.1",
-    "yaml": "^2.5.1"
+    "yaml": "^2.5.1",
+    "@babel/plugin-transform-class-static-block": "7.26.0"
```

Comment:
To resolve the package build error
<img width="1099" alt="Screenshot 2025-02-14 at 3 16 40 PM" src="https://github.com/user-attachments/assets/4b0380a4-813c-4a50-a16e-57da6c3e4c35" />